spacy/pipeline/dep_parser.pyx

```diff
@@ -132,7 +132,7 @@ cdef class DependencyParser(Parser):
             labeller.model.set_dim("nO", len(self.labels))
             if labeller.model.has_ref("output_layer"):
                 labeller.model.get_ref("output_layer").set_dim("nO", len(self.labels))
-            labeller.begin_training(get_examples, pipeline=pipeline, sgd=sgd)
+            labeller.initialize(get_examples, pipeline=pipeline)
```

Comment:
I guess `init_multitask_objectives` should get `nlp` as input so it can call `labeller.initialize(get_examples, nlp=nlp)`

Reply:
(same comment for ner.pyx)